recipes/photutils/meta.yaml

```diff
@@ -20,12 +20,13 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.9.*  # [not py36]
```

Comment:
If this works with np18 you can use it for all python's. (I added np18 last week,)